tests/execution/graphql-conformance-002.md

```diff
@@ -0,0 +1,109 @@
+---
+skip: true
+---
+
+# Test complex nested query. Skipped because Tailcall does not send the whole query to the remote server. It sends a shallow version of the query.
```

Comment:
```suggestion
# Test complex nested query. 

TODO: Skipped because Tailcall does not send the whole query to the remote server. It sends a shallow version of the query.
```

Maybe format like this? In my opinion it's more visible and allows to provide more info